src/schemes/oauth2.ts

```diff
@@ -259,7 +259,7 @@ export class Oauth2Scheme<
     // Set Nonce Value if response_type contains id_token to mitigate Replay Attacks
     // More Info: https://openid.net/specs/openid-connect-core-1_0.html#NonceNotes
     // More Info: https://tools.ietf.org/html/draft-ietf-oauth-v2-threatmodel-06#section-4.6.2
-    if (opts.response_type.includes('token')) {
+    if (opts.response_type.includes('id_token')) {
```

Comment:
Are you sure about this change? How do we know it won't break anyone else's setup?

Reply:
I can't be 100% sure of course, I have seen some issues describing PR #709 was broken, after changing to id_token it worked for me. On Stackoverflow was also something about it, and in Issue #970 as well.

Reply:
What about lowering the risk by checking for either value?

Reply:
I'll have a look at it later, I'll try to fix the checks of PR #1483 first. 😂

Reply:
Sounds good -- let me know if you get stuck. Thanks again for all your help!

Reply:
I've split this out into its own PR: https://github.com/nuxt-community/auth-module/pull/1532